server/tso/allocator_manager.go

```diff
@@ -328,19 +313,20 @@ func (am *AllocatorManager) campaignAllocatorLeader(loopCtx context.Context, all
 func (am *AllocatorManager) AllocatorDaemon(serverCtx context.Context) {
 	tsTicker := time.NewTicker(am.updatePhysicalInterval)
 	defer tsTicker.Stop()
-	checkerTicker := time.NewTicker(checkAllocatorStep)
+	patrolTicker := time.NewTicker(patrolStep)
+	defer patrolTicker.Stop()
+	checkerTicker := time.NewTicker(checkStep)
 	defer checkerTicker.Stop()
-	priorityTicker := time.NewTicker(checkPriorityStep)
-	defer priorityTicker.Stop()
 
 	for {
 		select {
 		case <-tsTicker.C:
 			am.allocatorUpdater()
-		case <-checkerTicker.C:
+		case <-patrolTicker.C:
 			am.allocatorPatroller(serverCtx)
-		case <-priorityTicker.C:
-			am.priorityChecker()
+		case <-checkerTicker.C:
+			am.ClusterDCLocationChecker()
```

Comment:
Why do we need to check it periodically?

Reply:
In case there are new DC nodes added into or removed from the cluster.